test/integration.test.js

```diff
@@ -40,7 +40,9 @@ describe('loopback application', function() {
       loopback.ACL.attachTo(db);
       loopback.User.hasMany(loopback.AccessToken, { as: 'accessTokens' });
 
-      var Streamer = app.model('Streamer', { dataSource: 'db' });
+      var Streamer = app.registry.createModel('Streamer');
+      // attach model to app and datasource
```

Comment:
I think this comment is not needed, can we remove it? I mean we are following this pattern (createModel + model) everywhere else, why to comment in this place only?